src/components/signup/AccountSetup.tsx

```diff
@@ -0,0 +1,234 @@
+import { useState } from 'react';
+import styled from 'styled-components';
+import VisibleIcon from '@svg/visible-icon.svg';
+import UnvisibleIcon from '@svg/unvisible-icon.svg';
+import BottomBtn from '@/components/common/BottomBtn';
+import { SignupFunnel } from '@/types/Funnel';
+
+const AccountSetup = (props: SignupFunnel) => {
+  const { signupInfo, setSignupInfo, toNextStep } = props;
+  const [isPWOpen, setIsPWOpen] = useState(false);
+  const [PWCheck, setPWCheck] = useState('');
+  const [isValid, setIsValid] = useState({
+    email: true,
+    password: true,
+    check: true,
+  });
+
+  const validate = (type: string, value: string) => {
+    const regexEmail =
+      /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
+    const regexPW = /^(?=.*[a-z])(?=.*[0-9]).{8,}$/;
+    type === 'email' &&
+      setIsValid({ ...isValid, email: regexEmail.test(value) });
+    type === 'password' &&
+      setIsValid({ ...isValid, password: regexPW.test(value) });
+    type === 'check' &&
+      setIsValid({ ...isValid, check: signupInfo.userPassword === value });
+  };
+
+  const handleInput = (type: string, value: string) => {
+    type === 'email'
+      ? setSignupInfo({ ...signupInfo, userEmail: value })
+      : type === 'password'
+        ? setSignupInfo({ ...signupInfo, userPassword: value })
+        : setPWCheck(value);
```

Comment:
이 부분 제가 쓴 코드 같긴 하지만.. `validate()` 처럼 아래와 같이 바꾸는게 좋을 것 같네요.
```suggestion
    type === 'email' && setSignupInfo({ ...signupInfo, userEmail: value })
    type === 'password' && setSignupInfo({ ...signupInfo, userPassword: value })
    type === 'pwCheck' && setPWCheck(value)
```